cairo/ethereum/cancun/fork.cairo

```diff
@@ -687,9 +687,11 @@ func check_transaction{
         let blob_gas_price = calculate_blob_gas_price(excess_blob_gas);
         let max_fee_per_blob_gas_u256 = tx.value.blob_transaction.value.max_fee_per_blob_gas;
         let max_fee_per_blob_gas_uint = U256_to_Uint(max_fee_per_blob_gas_u256);
-        let blob_gas_price_valid = is_le(max_fee_per_blob_gas_uint.value, blob_gas_price.value - 1);
+        let blob_gas_price_invalid = is_le(
+            max_fee_per_blob_gas_uint.value, blob_gas_price.value - 1
+        );
         with_attr error_message("InvalidBlock") {
-            assert blob_gas_price_valid = 1;
+            assert blob_gas_price_invalid = 0;
         }
```

Comment:
```suggestion
        if (blob_gas_price_invalid != 0){
            raise('InvalidBlock');
        }
```

should we migrate to?

Reply:
Wasn't it for those kind of assert 0 = 1 ?
Here the coverage si ok because other lines are reachable